packages/slider/src/spectrum-slider.css

```diff
@@ -408,278 +339,221 @@ governing permissions and limitations under the License.
+    );
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
     border-top-right-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
     border-bottom-left-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
     border-bottom-right-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
-    border-top-right-radius: 0; /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-top-right-radius: 0;
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
-    border-top-left-radius: 0; /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-top-left-radius: 0;
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
-    border-bottom-right-radius: 0; /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-bottom-right-radius: 0;
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
-    border-bottom-left-radius: 0; /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
-}
-:host([dir='ltr'][variant='range']) [dir='ltr'] .track,
-:host([dir='ltr'][variant='range']) [dir='rtl'] .track {
-    left: auto; /* [dir=ltr"] [dir="ltr] .spectrum-Slider--range .spectrum-Slider-track,
-   * [dir=ltr"] [dir="rtl] 
-    .spectrum-Slider--range .spectrum-Slider-track */
-}
-:host([dir='ltr'][variant='range']) [dir='rtl'] .track,
-:host([dir='rtl'][variant='range']) [dir='rtl'] .track {
-    right: auto; /* [dir=rtl"] [dir="ltr] .spectrum-Slider--range .spectrum-Slider-track,
-   * [dir=rtl"] [dir="rtl] 
-    .spectrum-Slider--range .spectrum-Slider-track */
-}
-:host([dir='ltr'][variant='range']) [dir='ltr'] .track,
-:host([dir='ltr'][variant='range']) [dir='rtl'] .track {
-    right: auto; /* [dir=ltr"] [dir="ltr] .spectrum-Slider--range .spectrum-Slider-track,
-   * [dir=ltr"] [dir="rtl] 
-    .spectrum-Slider--range .spectrum-Slider-track */
-}
-:host([dir='ltr'][variant='range']) [dir='rtl'] .track,
-:host([dir='rtl'][variant='range']) [dir='rtl'] .track {
-    left: auto; /* [dir=rtl"] [dir="ltr] .spectrum-Slider--range .spectrum-Slider-track,
-   * [dir=rtl"] [dir="rtl] 
-    .spectrum-Slider--range .spectrum-Slider-track */
```

Comment:
These get transformed weirdly.

Reply:
This forces them to apply, when previously they could never apply. Could be excluded, IIUC.